assets/src/legacy/filter.js

```diff
@@ -278,10 +278,22 @@ var lizLayerFilterTool = function () {
 
             // Get the HTML form element for the numeric field type
             function numericFormInput(field_item) {
```

Comment:
On IRC, I understood you would take care of `min_date` => `start_field` and `max_date` => `end_field`. These keys might be merged soon on my part.

It will avoid this screenshot : 
![ksnip_20220810-164839](https://user-images.githubusercontent.com/1609292/183933562-bc057cee-4025-41a9-8615-5f759b01d23d.png)